fe/fe-core/src/main/java/com/starrocks/catalog/Table.java

```diff
@@ -206,6 +207,8 @@ public static TableType deserialize(String serializedName) {
     @SerializedName(value = "mvs")
     protected Set<MvId> relatedMaterializedViews;
 
+    protected Optional<Long> snapshotId = Optional.empty();
```

Comment:
Is all other table format has long type snapshot id?

Reply:
The version id type of  known table format which can use `version as of`  are Long.